pipeline_dp/pipeline_operations.py

```diff
@@ -349,3 +346,213 @@ def count_per_element(self, col, stage_name: typing.Optional[str] = None):
 
     def reduce_accumulators_per_key(self, col, stage_name: str = None):
         return self.map_values(self.group_by_key(col), accumulator.merge)
+
+
+class Sentinel(Enum):
+    EOI = "EndOfIterations"
```

Comment:
Why do we need Sentinel? If it's neccassary please added explanation in comments

Reply:
It was an artifact of previous (unsuccessful) attempts of using `pool.imap_unordered`. Now it's not needed, I removed it.